### PR TITLE
Fixed block slots

### PR DESCRIPTION
**Pull request by srowhani**
#major#

# CHANGELOG

Upgraded to `ember-block-slots` 1.0 and adjusted the interface for the logo and actions

### CODE REVIEW

**Comment by sglanzer** [comment]
We don't have any README samples on using the slots, do we?

**Comment by srowhani** [comment]
@sglanzer Nope

File: addon/templates/components/frost-navigation.hbs [hunk 1]
```diff
@@ -4,11 +4,14 @@
     {{yield}}
   {{/yield-slot}}
   {{#each categories as |category|}}
-    {{nav-category
-      icon=category.icon
-      name=category.name
-      pack=category.pack
-    }}
+    {{#if category.visible}}
+      {{log category}}
```
Comment (sglanzer) [inline addon/templates/components/frost-navigation.hbs]: Missed log

File: addon/templates/components/frost-navigation.hbs [hunk 2]
```diff
@@ -4,11 +4,14 @@
     {{yield}}
   {{/yield-slot}}
   {{#each categories as |category|}}
-    {{nav-category
-      icon=category.icon
-      name=category.name
-      pack=category.pack
-    }}
+    {{#if category.visible}}
```
Comment (sglanzer) [inline addon/templates/components/frost-navigation.hbs]: What determines category visibliity?

Comment (srowhani) [inline addon/templates/components/frost-navigation.hbs]: That would be up to the app owner to decide. Probably permissions, or context

**Comment by sglanzer** [comment]
Approved

**Pull request merged**